### DISCUSSION TITLE
is there a command line option to move(look for) %LocalAppData%\Exult\ to the Exult installed directory?

### DISCUSSION THREAD

**Discussion by curvedlinee** [exult/exult#550] [General]
is there a command line option or a game menu option to move(look for) %LocalAppData%\Exult\ to the Exult installed directory like OpenXcom UFO?

**Comment by wench** [comment]
yes. Start Exult with a -p argument to run in portable mode using the startup directory instead of AppData\Exult on Windows. If the startup directory is the installed directory it will behave as you want. This is intended to allow running exult from portable media like a usb stick

**Comment by curvedlinee** [comment]
thanks very much,  it  was perfect

start "" Exult.exe -p --bg --nomenu

i changed the path in exult.cfg, it now is a drive directory name independent... more perfect
blackgate
 path
 .\Ultima 7 - Black Gate

is there a way to change(or implement it in the future) the Ultima fonts like the ones from westwood's kyrandia or lucas arts's monkey islands? thick fonts.

**Comment by wench** [comment]
Our modding tools can be used to create custom fonts but it is a very manual process, there are no current plans for automatic font replacements.

**Reply by KnightCaptainU7** [reply]
Wouldn't they be a file replacement like the shapes.vga?
